packages/package.json

```diff
@@ -17,7 +17,7 @@
     "prebuild": "rm -Rf */dist",
     "build": "webpack"
   },
-  "devDependencies": {
+  "dependencies": {
```

Comment:
We don't really need those to be `devDependencies`, since we don't need to differentiate dev and non-dev deps...

Reply:
It does not change the `npm build` ?

Reply:
Oh no, the deps are available in the same way.